pkgs/shells/pdksh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  name = "pdksh-${version}";
+  version = "5.2.14";
+
+  src = fetchurl {
+    url = "http://www.cs.mun.ca/~michael/pdksh/files/${name}.tar.gz";
+    sha256 = "19mzkfmj3ms8sp9m8f6fyba0am6fxnr538xw6b3fv4gja3fvq5db";
+  };
+
+  buildInputs = [ ];
+
+  # Traditionally, PDKSH is used as "pdksh" rather than "ksh"
+  configureFlags = [ "--program-prefix=pd" ];
+
+  patches = [
```

Comment:
Regularly I watch AUR or regular Archlinux PKGBUILDs in order to be more acquainted with the software packaging. In AUR, there is a big fat patch, and I have split them in my testing. But the official homepage has a patch too:
http://www.cs.mun.ca/~michael/pdksh/files/

pdksh is a bit outdated (2008?), so I think it is more secure and sensible if we maintain our particular set.